package.json

```diff
@@ -21,14 +21,16 @@
   },
   "homepage": "https://github.com/northcoders/be-nc-news#readme",
   "devDependencies": {
+    "husky": "^7.0.0",
     "jest": "^27.5.1",
     "jest-extended": "^2.0.0",
-    "pg-format": "^1.0.4",
-    "husky": "^7.0.0"
+    "pg-format": "^1.0.4"
   },
   "dependencies": {
     "dotenv": "^16.0.0",
-    "pg": "^8.7.3"
+    "express": "^4.18.1",
+    "pg": "^8.7.3",
+    "supertest": "^6.2.4"
```

Comment:
supertest is a dev dependency so just move this into the devDependencies object :)